pkg/pipelines/bootstrap.go

```diff
@@ -536,3 +536,20 @@ func getResourceFiles(res res.Resources) []string {
 	sort.Strings(files)
 	return files
 }
+
+//check if the file exists or not
+func CheckFileExists(fs afero.Fs, dockerConfigJSONFilename string) (string, error) {
+	if dockerConfigJSONFilename == "" {
+		return "", errors.New("failed to generate path to file: --dockerconfigjson flag is not provided")
+	}
+	authJSONPath, err := homedir.Expand(dockerConfigJSONFilename)
+	if err != nil {
+		return "", fmt.Errorf("failed to generate path to file: %v", err)
+	}
+	f, err := fs.Open(authJSONPath)
```

Comment:
This should use `https://golang.org/pkg/os/#Stat` to check if the file exists (and is not a directory) in combination with https://golang.org/pkg/os/#IsNotExist

Reply:
This has been incoprorated.